src/app/event-create/event-create.component.ts

```diff
@@ -36,16 +36,23 @@ export class EventCreateComponent implements OnInit {
   onSubmit() {
     console.warn(this.eventForm.value);
 
-    this.eventService.createOrUpdate({
+    let event = {
       eventId: '',
       name: this.eventForm.get('name').value,
       description: this.eventForm.get('description').value,
-      eventStart: this.eventForm.get('eventStart').value,
-      eventEnd: this.eventForm.get('eventEnd').value,
-      deadlineRVSP: this.eventForm.get('deadlineRVSP').value,
+      eventStart: this.dateToMilliseconds(this.eventForm.get('eventStart').value),
+      eventEnd: this.dateToMilliseconds(this.eventForm.get('eventEnd').value),
+      deadlineRVSP: this.dateToMilliseconds(this.eventForm.get('deadlineRVSP').value),
       location: this.eventForm.get('location').value,
       organizer: this.eventForm.get('organizer').value
```

Comment:
Could make a private function for returning the value of a control in the eventForm, so that you dont have to type this.eventForm.get().value everytime.